CalibCalorimetry/EcalLaserAnalyzer/plugins/EcalStatusAnalyzer.h

```diff
@@ -4,18 +4,20 @@
 #include <map>
 
 #include <memory>
-#include <FWCore/Framework/interface/EDAnalyzer.h>
+#include <FWCore/Framework/interface/stream/EDAnalyzer.h>
 
 class Timestamp;
 
-class EcalStatusAnalyzer : public edm::EDAnalyzer {
+class EcalStatusAnalyzer : public edm::stream::EDAnalyzer<> {
 public:
   explicit EcalStatusAnalyzer(const edm::ParameterSet& iConfig);
   ~EcalStatusAnalyzer() override;
 
-  void analyze(const edm::Event& e, const edm::EventSetup& c) override;
-  void beginJob() override;
-  void endJob() override;
+  void analyze(const edm::Event& e, const edm::EventSetup& c);
+  //void analyze(edm::StreamID, const edm::Event& e, const edm::EventSetup& c);
+  //void analyze(edm::StreamID, const edm::Event& e, const edm::EventSetup& c) const override ;
+  void beginJob();
```

Comment:
If you have to remove the `override` keyword this means that the base class does not have the function and therefore the CMSSW framework does never call these functions for this module type. `beginJob()` exists for `one` and `global` and should have the `override` keyword for those types.

Reply:
Placed override back in.